File: app/src/main/java/com/porkandlager/travelonbudget/wires/models/beans/FlightSearch.java
package com.porkandlager.travelonbudget.wires.models.beans;

import com.google.gson.annotations.SerializedName;

/**
 * Created by tista on 11/12/16.
 */

public class FlightSearch {

    @SerializedName("airports")
    private Airports airports;

    @SerializedName("referral_link")
    private String referralLink;

    @SerializedName("inbound")
    private Quote inbound;

    @SerializedName("outbound")
    private Quote outbound;

    @SerializedName("cheapest")
    private int cheapestPrice;

    @SerializedName("dates")
    private FlightDates flightDates;

    @SerializedName("contents")
    private Content content;

    public Airports getAirports() {
        return airports;
    }

    public String getReferralLink() {
        return referralLink;
    }

    public Quote getInbound() {
        return inbound;
    }

    public Quote getOutbound() {
        return outbound;
    }

    public int getCheapestPrice() {
        return cheapestPrice;
    }

    public Content getContent() {
        return content;
    }
}

File: app/src/main/java/com/porkandlager/travelonbudget/wires/models/beans/FlightSearch.kt
package com.porkandlager.travelonbudget.wires.models.beans

import com.google.gson.annotations.SerializedName

/**
 * Created by tista on 11/12/16.
 */

class FlightSearch {

    @SerializedName("airports")
    var airports: Airports? = null

    @SerializedName("referral_link")
    var referralLink: String? = null

    @SerializedName("inbound")
    var inbound: Quote? = null

    @SerializedName("outbound")
    var outbound: Quote? = null

    @SerializedName("cheapest")
    var cheapestPrice: Int = 0

    @SerializedName("dates")
    var flightDates: FlightDates? = null

    @SerializedName("contents")
    var content: Content? = null

}
